org.eclipse.wildwebdeveloper.embedder.node.linux.x86_64/pom.xml

```diff
@@ -7,7 +7,7 @@
 		<version>0.5.0-SNAPSHOT</version>
 	</parent>
 	<packaging>eclipse-plugin</packaging>
-	<version>0.1.0-SNAPSHOT</version>
+	<version>0.1.1-SNAPSHOT</version>
```

Comment:
Is the version bump here really necessary?

Reply:
Result without it: https://ci.eclipse.org/wildwebdeveloper/job/Wildwebdeveloper/job/PR-489/1/console

Reply:
OK, I see that `<jgit.ignore>` isn't set to `pom.xml`, that explains why this is required.
However, we'll need to find ways to avoid that, as it will then force such change to redistribute the bundle to all consumers for no good reason.

Reply:
setting jgit.ignore doesn't seem to have helped. I think of bumping the version this time together with jgit.ignore addition. It will also show us how good update of node fragment works.